webservices/resources/elections.py

```diff
@@ -91,8 +91,14 @@ def _get_elections(self, kwargs):
             CandidateHistory.candidate_inactive == False,  # noqa
             # TODO(jmcarp) Revert after #1271 is resolved
             sa.or_(
-                CandidateHistory.district == None,  # noqa
-                CandidateHistory.district != '99',
+                #This line was causing some semi-duplicate entries
+                #an example here: https://beta.fec.gov/data/elections/?cycle=2016&state=AK
+                #young shows up once. In one of those entries the district is null.  I don't know
+                #if this was to counter a bug or some incomplete data.  But I would say we don't
+                #want candidates with null districts showing up.  Thoughts?  The TODO directly
+                #above is also indicator of some prior inconsistent data, is this still the case?
+                #CandidateHistory.district == None,  # noqa
+                CandidateHistory.district != '99'
```

Comment:
Correct me if I am wrong, but I think that since we wrote this we have switched the underlying data source I don't think we need this line, the 99 was a warehouse problem. We are using`disclosure.cand_valid_fec_yr` which is better there are some inconsistency in the numbers but it's things like 07 and 7, but that shouldn't be too bad for our purposes. 

As far as the Null values, @PaulClark2 @jwchumley there are 59 records, all house members that are missing districts. Is that something we can fix in `disclosure.cand_valid_fec_yr`? I think the more we can keep this logic simplified, the better.